### PR TITLE
Remove unneeded strings from firefox/campaign.lang

### PR DESCRIPTION
**Pull request by alexgibson**
Removes unneeded strings from `firefox/campaign.lang`, leaving only the strings required for this page: https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/campaign/

### CODE REVIEW

File: en-US/firefox/campaign.lang [hunk 1]
```diff
@@ -1,28 +1,7 @@
...
-
-;You’re using a pre-release version of Firefox.
-You’re using a pre-release version of Firefox.
-
-
-;You’re using an insecure, outdated operating system <a href="%(url)s">no longer supported by Firefox</a>.
-You’re using an insecure, outdated operating system <a href="%(url)s">no longer supported by Firefox</a>.
-
-
 ## TAG: firefox_quantum_new_headline
```
Comment (flodolo) [inline en-US/firefox/campaign.lang]: I see an error locally:

```
The following tags are bound to strings but not declared at the beginning of the file:

    firefox_quantum_new_headline
``` 

In other words: you have a tag associated to a string, but there's no tag declared at the beginning of the file. I'll clean it up.

**Comment by flodolo** [comment]
I kept separate commits to give you an idea of what needs to happen:
* Removed the tags
* Removed ACTIVE tag from all files. You changed the file, and the tags, so we don't really know which locales should be enabled with the new content
* Run lang_update on the file for all locales

**Comment by alexgibson** [comment]
Thanks @flodolo - where do these scripts live, and do we have docs for them?

**Comment by flodolo** [comment]
Scripts are in this project: https://github.com/mozilla-l10n/langchecker/
Docs are here: https://mozilla-l10n.github.io/documentation/products/mozilla_org/

@peiying2 is working on a summary document for you

**Pull request merged**